### PR TITLE
feat(ignore): always ignore genignore

### LINKED ISSUE #102: clean up .genignore automagically
the current impl of `.genignore` requires that you explicitly add the `.genignore` file in order for it to be cleaned up. as @killercup pointed out when i spoke to him, this is pretty surprising. because the `.genignore` file should always be cleaned up, let's automagically do so, and not require users to list it in the `.genignore` file itself.

### PR DESCRIPTION
**Pull request by ashleygwilliams**
fixes #102

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
lol too bad it didn't work tho. gonna sort this today as it's blocking the release that i'm a week late on now :(

File: src/ignoreme.rs [hunk 1]
```diff
@@ -36,11 +36,7 @@ fn get_ignored(location: &PathBuf) -> Option<Vec<PathBuf>> {
     for x in all_set.difference(&ign_set) {
         output.push(x.to_owned());
     }
-    if output.is_empty() {
-        None
-    } else {
-        Some(output)
-    }
+    Some(output)
```
Comment (k0pernicus) [inline src/ignoreme.rs]: I checked the file and didn't found any clue about an Option type other that `Some(output)`.
If I did not made any mistake in reading the code, why do we return an `Option` type if it's always `Some(output)`... ?

Comment (ashleygwilliams) [reply src/ignoreme.rs]: yeah i think that's a good call, previously it returned an option because there may be nothing in Output, i'll change it

**Comment by k0pernicus** [comment]
Ok to me :ok_hand:

**Pull request merged**